meal-planner/src/components/MealSelect.js

```diff
@@ -0,0 +1,49 @@
+import styled from 'styled-components'
+
+export default function MealSelect({ key, onClick }) {
+    const meals = [
+        'Käsestulle',
+        'Kartoffelsalat',
+        'Gemüselasagne',
+        'Mayo',
+        'Currywurst',
+        'Linsensuppe',
+        'Frikassee',
+        'Currywurst',
+        'Katzenfutter',
+    ]
+
+    function onMealClick(meal) {
+        console.log(meal)
```

Comment:
Is it a dummy for upcomming functionality? Not necessary to already implement it in the code.